pkg/runner/cluster_k8s.go

```diff
@@ -341,8 +342,8 @@ func (c *ClusterK8sRunner) Run(ctx context.Context, input *api.RunInput, ow *rpc
 						return err
 					}
 
-					fmt.Print(logs)
```

Comment:
Previously we were just printing the last 2 log lines from every container on the daemon process, as a shortcut.

Obviously this doesn't work anymore, as we are generally interested in all logs, and also want to propagate them back to the user.